README.md

```diff
@@ -2,9 +2,43 @@
 
 ## Tech Stack
 
-Framework: Spring Boot
-Language: Java 17
-Build Tool: Maven
+Framework : 
+* Core
+    * Spring
+        * Spring Boot 3
+        * Spring Boot Test (Junit)
+        * Spring Boot Dev Tools
+        * Spring Security
+        * Spring Web
+        * Spring Boot JOOQ
+        * Spring Boot Actuator
+    * Spring Data
+        * Spring Data JPA
+        * Spring Data REST
+    * Cache
+        * Spring Cache
+
+Databases :
+* MYSQL
+* H2 Database
+
+Technologies :
+* Lombok
+* Test Containers
+
+Language : 
+* Java 17
+
+Build Tool : 
+* Maven
+
+Software Development Process :
+* TDD
+* Agile Scrum
+
+Version Control :
+* Git
+
```

Comment:
Buradaki teknolojilere ve süreçlere @Afet-Yonetim-Sistemi/ays-be ile birlikte karar verelim, eklemesi olan vardır, sonrasında mergeleyelim

Reply:
pom.xml bakıp o şekilde yapmıştım.

Reply:
@Rapter1990 tamamdır, conflict var onu çözdükten sonra mergeleyelim 👍🏼

Reply:
@agitrubard biraz önce commit merge ettim.